core/templates/pages/admin-page/admin-page.directive.ts

```diff
@@ -91,12 +91,6 @@ angular.module('oppia').directive('adminPage', [
               AdminRouterService.showTab($location.path().replace('/', '#'));
             });
             ctrl.userEmail = '';
```

Comment:
Can you please confirm whether this is not used in the template?

Reply:
I'm sorry there was a confusion that the user email is not being used in the template